test/e2e/basic_test.go

```diff
@@ -82,10 +82,10 @@ func TestPauseControl(t *testing.T) {
 	if err := e2eutil.KillMembers(f.KubeClient, f.Namespace, names[0]); err != nil {
 		t.Fatal(err)
 	}
-	if _, err := e2eutil.WaitUntilSizeReached(t, f.KubeClient, 2, 10*time.Second, testEtcd); err != nil {
+	if _, err := e2eutil.WaitUntilPodSizeReached(t, f.KubeClient, 2, 10*time.Second, testEtcd); err != nil {
```

Comment:
@hongchaodeng The name has not changed. All the tests are still using `WaitUntilSizeReached` for which only the implementation has changed.
Over here `WaitUntilSizeReached()` with the new implementation cannot be used because the operator is paused and the TPR will never be updated to show the size reduce to 2. So we use `WaitUntilPodSizeReached` which has the original implementation of checking the number of pods directly.

The tests for `TestPauseControl ` and `TestRestartOperator ` will not pass if this is kept to `WaitUntilSizeReached()`.